### PR TITLE
[Web] animation & dialog UI

### PR DESCRIPTION
**Pull request by michian**

### CODE REVIEW

File: src/components/Button/ActionButton.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Box } from '@material-ui/core';
+import { isPendingTxn, txnButtonText } from '../../store/slices/pending-txns-slice';
+import Loading from '../Loader';
+
+function ActionButton(props: any) {
```
Comment (otterclamking) [inline src/components/Button/ActionButton.tsx]: need to define props interface

**Review by otterclamking — CHANGES_REQUESTED** [review]

**Comment by otterclamking** [comment]
Hey @michian The dialog close button is wrong on Firefox 95.0b12 (64-bit).

CleanShot 2021-12-05 at 12 34 00@2x

About the success dialog, it works for the bonding. I suggest separating the bond/stake integration into two PRs. 

After fixed this, we could merge the bond part first.

**Comment by michian** [comment]
Hey @otterclamking, I pushed the changes on the close button for firefox.
image


About the stake/unstake not working, I can't separate the pr because I changed a function used by all action buttons.

I tried to stake/unstake with my own wallet and It's working fine. No changes on my side.
image

**Comment by michian** [comment]
For some reason stake/unstake in getting stuck when using local node but works fine when using polygon network.

**Review by otterclamking — APPROVED** [review]

**Pull request merged**